src/test/txpackage_tests.cpp

```diff
@@ -283,6 +283,14 @@ BOOST_AUTO_TEST_CASE(noncontextual_package_tests)
         BOOST_CHECK(GetPackageHash({tx_parent}) != GetPackageHash({tx_child}));
         BOOST_CHECK(GetPackageHash({tx_child, tx_child}) != GetPackageHash({tx_child}));
         BOOST_CHECK(GetPackageHash({tx_child, tx_parent}) != GetPackageHash({tx_child, tx_child}));
+
+        // IsChildWithParents* can also be a singleton
+        BOOST_CHECK(IsChildWithParents({tx_parent}));
+        BOOST_CHECK(IsChildWithParentsTree({tx_parent}));
```

Comment:
nit: we require exactly one child, so I think using `tx_parent` is unnecessarily confusing
```suggestion
        BOOST_CHECK(IsChildWithParents({tx_child}));
        BOOST_CHECK(IsChildWithParentsTree({tx_child}));
```

Reply:
sure done